postcss.config.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-const { media_breakpoints } = require("./static/js/css_variables.js");
+const {media_breakpoints} = require("./static/js/css_variables.js");
```

Comment:
It looks like this is among various lint fixes that belong in the previous commit.

Reply:
fixed